client-app/shared/static-content/components/index.ts

```diff
@@ -2,23 +2,25 @@ import { Component } from "vue";
 import TextBlock from "./text-block.vue";
 import ImageBlock from "./image-block.vue";
 import ProductInfo from "./product-info.vue";
-import SideImage from "./side-image.vue";
+import CallToActionWithImage from "./call-to-action-with-image.vue";
 import PromoText from "./promo-text.vue";
-import ColumnImages from "./column-images.vue";
-import DarkPromoText from "./dark-promo-text.vue";
+import Features from "./features.vue";
+import CallToAction from "./call-to-action.vue";
 import SubscribeForm from "./subscribe-form.vue";
 import ProductsBlock from "./products-block.vue";
+import TitleBlock from "./title-block.vue";
 
 const templateBlocks: { [key: string]: Component } = {
   "text-block": TextBlock,
   "image-block": ImageBlock,
   "product-info": ProductInfo,
-  "side-image": SideImage,
+  "call-to-action-with-image": CallToActionWithImage,
   "promo-text": PromoText,
-  "column-images": ColumnImages,
-  "dark-promo-text": DarkPromoText,
+  features: Features,
+  "call-to-action": CallToAction,
   "subscribe-form": SubscribeForm,
   products: ProductsBlock,
+  title: TitleBlock,
```

Comment:
maybe add postfix "-block" to all of them

Reply:
added